### PR TITLE
Include Sift client-side snippet on all page-loads, if configured (PLATFORM-2978)

### PR DESCRIPTION
**Pull request by joeyAghion**
This adds the [Sift client-side snippet](https://sift.com/developers/docs/ruby/javascript-api/overview) to all page views, as we did for Gravity (https://github.com/artsy/gravity/pull/13877 🔒) and Volt (https://github.com/artsy/volt/pull/4972 🔒).

Depends on an already-performed migration to configure the new `SIFT_BEACON_KEY` (sandbox) value:

```
hokusai [staging|production] env set SIFT_BEACON_KEY=...
```

Still to-do:
* Ideally, client-side navigations call `_sift.push(['_trackPageview']);` as per [the docs](https://support.sift.com/hc/en-us/articles/208370598-Using-the-JavaScript-snippet-in-a-Single-Page-App).

https://artsyproduct.atlassian.net/browse/PLATFORM-2978

### CODE REVIEW

**Comment by joeyAghion** [comment]
Updated to also track client-side-navigated pages.

File: src/v2/Artsy/Router/Boot.tsx [hunk 1]
```diff
@@ -78,6 +79,7 @@ export const Boot = track(null, {
                       <GlobalStyles />
                       <FlashMessage />
                       <FocusVisible />
+                      <SiftContainer />
```
Comment (icirellik) [inline src/v2/Artsy/Router/Boot.tsx:82]: ✨ lazy loading!

File: src/v2/Utils/SiftContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    if (sd.THIRD_PARTIES_DISABLED || !sd.SIFT_BEACON_KEY) {
+      return
+    }
+
+    if (document.getElementById("sift-beacon")) {
+      return
+    }
+    const script = document.createElement("script")
+    script.id = "sift-beacon"
+    script.async = true
```
Comment (icirellik) [inline src/v2/Utils/SiftContainer.tsx:16]: I'm not sure `async` or `defer` do anything for inline text. Still, it's safe to leave it.

**Review by icirellik — APPROVED** [review]
Looks awesome to me. Nice and simple and shouldn't cause any performance regressions.

File: src/v2/Utils/SiftContainer.tsx (hunk 2 above)
Comment (joeyAghion) [reply src/v2/Utils/SiftContainer.tsx:16]: This reminds me of a follow-up question: should we set `async` or `defer` on the injected `script` element?

**Pull request merged**